Resources/Prototypes/Entities/Structures/Power/Generation/generators.yml

```diff
@@ -247,3 +264,22 @@
       - state: rtg_glow
   - type: RadiationSource # ideally only when opened.
     intensity: 2
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 50
+      behaviors:
+      - !type:PlaySoundBehavior
+        sound:
+          path: /Audio/Effects/metalbreak.ogg
+      - !type:SpawnEntitiesBehavior
+        spawn:
+          SheetSteel1:
+            min: 5
```

Comment:
min should be something less than 5, this means it won't consistently give full resources (since it's being destroyed). Same for Uranium. Otherwise very nice changes

Reply:
done